docs/usage.rst

```diff
@@ -47,7 +47,7 @@ Backup Destination
 The ``[destination]`` section specifies where to store backup copies.
 
 ``backup_destination`` can be either ``ssh`` (if you want to store backups on a remote SSH server),
-or ``s3`` (if you want to store backups in Amazon S3), or ``gsc`` (if the backup should be stored in Google Cloud).
+``s3`` (if you want to store backups in Amazon S3), ``az`` (if the backup should be stored in Azure Blob Storage), or ``gsc`` (if the backup should be stored in Google Cloud).
```

Comment:
I think there's an original typo here -- in that it should read gcs and not gsc

Reply:
resolved